### PR TITLE
[Demo] Improve BLE demo stability

### PR DESCRIPTION
**Pull request by jimmy-huang**
It seems that the temperature value change update is causing random
update issue and dropped connection, changed the demo code to
only send updates if a value change is detected.

Also increase the interval between updates to a minimum of 2sec.

Signed-off-by: Jimmy Huang <jimmy.huang@intel.com>

### CODE REVIEW

File: arc/src/main.c [hunk 1]
```diff
@@ -13,9 +13,9 @@
 #include "zjs_common.h"
 #include "zjs_ipm.h"
 
-#define QUEUE_SIZE      10  // max incoming message can handle
-#define SLEEP_TICKS      1  // sleep time in cpu ticks
-#define UPDATE_INTERVAL 50  // interval in between notifications
+#define QUEUE_SIZE       10  // max incoming message can handle
+#define SLEEP_TICKS       1  // sleep time in cpu ticks
+#define UPDATE_INTERVAL 200  // interval in between notifications
```
Comment (kenchris) [inline arc/src/main.c]: State in what unit? I would not guess 200 to be very 2 seconds

File: samples/WebBluetoothDemo.js [hunk 2]
```diff
@@ -148,9 +148,8 @@ ble.on('stateChange', function(state) {
 });
 
 ble.on('advertisingStart', function(error) {
-    print('advertisingStart: ' + (error ? error : 'success'));
-
     if (error) {
+        print("Failed to advertise physical web");
```
Comment (kenchris) [inline samples/WebBluetoothDemo.js]: ... Physical Web URL.

File: samples/WebBluetoothDemo.js [hunk 3]
```diff
@@ -163,25 +162,31 @@ ble.on('advertisingStart', function(error) {
...
+    var lastTemp = 0;
 
     tmp36.on("change", function(data) {
         var voltage = (data / 4096.0) * 3.3;
         var celsius = (voltage - 0.5) * 100 + 0.5;
         celsius = celsius | 0;
 
-        print("Temperature change: " + celsius + " degrees Celsius");
-        TemperatureCharacteristic.valueChange(celsius);
+        if (celsius != lastTemp) {
```
Comment (kenchris) [inline samples/WebBluetoothDemo.js]: !== ?

**Comment by jimmy-huang** [comment]
Updated PR based on @kenchris feedback

**Comment by poussa** [comment]
LGTM, based on the feedback from @kenchris this is improving the demo.

**Pull request merged**